src/highdicom/map/content.py

```diff
@@ -0,0 +1,120 @@
+from typing import Optional, Sequence, Tuple, Union
+
+from highdicom.sr.coding import CodedConcept
+from pydicom.dataset import Dataset
+from pydicom.sr.coding import Code
+
+
+class RealWorldValueMapping(Dataset):
```

Comment:
@dclunie could you kindly take a look at the implementation of the [Real World Value Mapping Item Macro](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.16.2.html#sect_C.7.6.16.2.11). In particular, I would appreciate if you could assess we interpreted *Real World Value First/Last Value Mapped* correctly.

In general, I wonder whether that functional group macro should be required, given that a relationship to real world values may not exist for many use cases.

The standard states

> The Items in the Real World Value Mapping Sequence (0040,9096) **may be used** to translate stored values into real world values **when there is such a relationship**

Making the usage `C` instead of `M` would simplify things quite a bit in my opinion.

Reply:
These are required and that is not going to change. This has been discussed extensively in the past.

Worst case, for dimensionless properties, an identity transformation over the full range of possible values is specified.

Reply:
Thanks for clarification @dclunie